Sources/Quick/QuickSpec.m

```diff
@@ -115,10 +115,23 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
     }
 
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
-                              example.name.qck_selectorName,
-                              sanitizedFileName,
-                              (long)example.callsite.line];
+    
+    NSString *originalName = example.name.qck_selectorName;
+    NSString *selectorName = originalName;
+    unsigned int i = 2;
```

Comment:
I'd used it in my first commit, but changed the variable type to silence a compiler warning. I updated to instead cast to `unsigned long` in the `stringWithFormat:` call.